thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/annotation/DetectionTag.java

```diff
@@ -23,5 +23,6 @@
   ALGORITHM_DETECTION,
   RULE_DETECTION,
   ALGORITHM_FILTER,
-  RULE_FILTER
+  RULE_FILTER,
+  CUSTOM
```

Comment:
The name "CUSTOM" is too general. Why not calling it "GROUPER"?

Reply:
Updated